presto-cli/pom.xml

```diff
@@ -14,6 +14,9 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <main-class>io.prestosql.cli.Presto</main-class>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
+        <air.check.skip-checkstyle>true</air.check.skip-checkstyle>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
```

Comment:
can we fix and remove these?

Reply:
Yes. I will. For this PR I just fixed all the namespace issues from the hackathon project.

Reply:
Removed 2 of those. Going to fix the style

Reply:
Removed all skips